doc/bridle/gs_nix.rst

```diff
@@ -0,0 +1,58 @@
+.. _gs_nix:
+
+Working with Bridle via Nix
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+An alternative way to develop with Bridle is through `Nix <https://nixos.org/>`_, a functional package
+managing system. This way, all dependencies can be installed using a single
+tool. Packages installed this way don't pollute your regular system, and can
+easily be removed later.
+
+Installing the Nix Package Manager
+**********************************
+
+To get started using Nix, you will need to install the package manager itself.
+Most distributions provide packages for installing it using their regular
+package manager (e.g. Ubuntu provides a ``nix`` package that can be installed
+using ``apt install nix``). If your distribution doesn't provide such a package, see the
+instructions on `nixos.org <https://nixos.org/download>` to get a Nix
+installation.
+
+Enabling Flakes
+***************
+
+Flakes are an experimental feature that enables fully hermetic and reproducible
+builds. To enable flakes, add
+
+.. parsed-literal::
+   :class: highlight
+
+   experimental-features = nix-command flakes
+
+to ``~/.config/nix/nix.conf`` (creating this file if it doesn't exist).
```

Comment:
**praise (non-blocking):** I would prefer:
"… to your private Nix configuration in ``~/.config/nix/nix.conf``…"

Reply:
I've adjusted this bit.